### PR TITLE
Win32: Use a dedicated background thread for each remote UI thread.

### PR DESCRIPTION
**Pull request by madewokherd**
This works independent of COM marshaling, which is not implemented in Mono.

### CODE REVIEW

File: xalia/Win32/AccessibleProvider.cs [hunk 1]
```diff
@@ -676,7 +677,7 @@ private async Task PollChildren()
                             break;
                     }
                     return result;
-                }, Tid);
+                }, CommandThreadPriority.Query);
```
Comment (Conmanx360) [inline xalia/Win32/AccessibleProvider.cs:680]: Was this supposed to be +1 originally and you've fixed it here, or should this be ` CommandThreadPriority.User` like the other `Tid` cases?

File: xalia/Win32/NonclientScrollProvider.cs [hunk 2]
```diff
@@ -287,7 +287,7 @@ private async Task RefreshScrollBarInfo()
                     if (!GetScrollBarInfo(Hwnd, Vertical ? OBJID_VSCROLL : OBJID_HSCROLL, ref _sbi))
                         throw new Win32Exception();
                     return _sbi;
-                }, Tid);
+                }, CommandThreadPriority.Query);
```
Comment (Conmanx360) [inline xalia/Win32/NonclientScrollProvider.cs:290]: Same case here.

File: xalia/Win32/NonclientScrollProvider.cs [hunk 3]
```diff
@@ -371,7 +371,7 @@ private async Task RefreshScrollInfo()
                         throw new Win32Exception();
 
                     return si;
-                }, Tid);
+                }, CommandThreadPriority.Query);
```
Comment (Conmanx360) [inline xalia/Win32/NonclientScrollProvider.cs:374]: Here too.

File: xalia/Win32/NonclientScrollProvider.cs [hunk 4]
```diff
@@ -430,7 +430,7 @@ public async Task<double> GetMinimumIncrementAsync(UiDomElement element)
                         throw new Win32Exception();
 
                     return CalculateMinimumIncrement(sbi, si);
-                }, Tid);
+                }, CommandThreadPriority.Query);
```
Comment (Conmanx360) [inline xalia/Win32/NonclientScrollProvider.cs:433]: And here. (I'm beginning to guess these were just incorrectly marked as `CommandThreadPriority.User` to begin with and fixed now)

**Comment by madewokherd** [comment]
I think most of those were mismarked, but GetMinimumIncrementAsync probably should be CommandThreadPriority.User, since that's used in adjust_scrollbars.

**Pull request merged**